Common/Data/Model/File.swift

```diff
@@ -10,12 +10,30 @@ import MobileCoreServices
 
 public final class File: NSManagedObject {
 
+    @objc public enum OwnerType: Int64 {
```

Comment:
Could we make `OwnerType` based on Strings?
```
enum OwnerType: String { ... }
````

Reply:
I could, though that add a little boilerplate because `String` enum aren't `@objc` compatible and the benefit seems really marginal.